codegenerator/cli/src/config_parsing/chain_helpers.rs

```diff
@@ -245,6 +228,25 @@ pub enum Network {
     // // Explorers:
     // // https://explorer.degen.tips/
     // Degen = 666666666,
+    #[subenum(HypersyncNetwork)]
+    Chiliz = 8888,
+    #[subenum(HypersyncNetwork)]
+    IncoGentryTestnet = 9090,
+    #[subenum(HypersyncNetwork)]
+    Zircuit = 48900,
+    #[subenum(HypersyncNetwork)]
+    MevCommit = 17864,
+    #[subenum(HypersyncNetwork)]
+    GaladrialDevnet = 696969,
+    #[subenum(HypersyncNetwork)]
+    SophonTestnet = 531050104,
+    #[subenum(HypersyncNetwork)]
+    KakarotSepolia = 1802203764,
+    #[subenum(HypersyncNetwork)]
+    BerachainBartio = 80084,
+    /// Still syncing
```

Comment:
```suggestion
    // // Still syncing
```

A typo, fixed in next PR.